### ISSUE TITLE
A few AVX tests failing when XSAVE is disabled at OS level

### ISSUE THREAD

**Issue by VSadov** [dotnet/runtime#47317]
While working on https://github.com/dotnet/runtime/pull/47212 , I did some testing with XSAVE disabled. (as in `bcdedit /set xsavedisable 1`)

One thing that I noticed is that a few clr tests failed, without any of my changes.
Most likely it is an issue with feature detection in the test harness. Everything else works fine and Libs tests were all passing.

Logging this just in case this is something unexpected or worth fixing.

**Comment by VSadov** [comment]
CC: @tannergooding

**Comment by tannergooding** [comment]
I'll take a look tomorrow. Do you happen to know which of the tests were failing?

CC. @jeffhandley

**Comment by VSadov** [comment]
@tannergooding I did not catch what tests failed exactly. I saw a few AVs in the output and it was all about vector types. 

Knowing that I have disabled XSAVE, I am pretty sure it is AVX related. Also knowing that everything works fine otherwise, which means regular feature detection is ok, it must be something with the tests or harness. 

It is probably not a high priority if it is test-only. Hard to see someone getting into this by accident.
